3p/environment.js

```diff
@@ -18,7 +18,7 @@ export function setInViewportForTesting(inV) {
 // Active intervals. Must be global, because people clear intervals
 // with clearInterval from a different window.
 const intervals = {};
-let intervalId = 1;
+let intervalId = 10000000;
```

Comment:
could you add a Note here explaining this for future readers of the code.

also could you add a github issue for a follow up and annotate the issue id here. like you said a proper solution is going to be a little more involved but we definitely want to fix this but this is a good patch for now.